frontend/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "bowtie",
+  "version": "1.0.0",
+  "description": "Schema validator app",
```

Comment:
Possibly copy the description from the `pyproject.toml` file here (or copy it and mention this is the UI for it).